apps/frontend/src/pages/chat/libs/components/buttons-controller/libs/components/suggestions-creation-options/libs/helpers/get-categories-with-three-lowest-scores.helper.ts

```diff
@@ -0,0 +1,12 @@
+import { NumericalValue } from "~/libs/enums/enums.js";
+import { type QuizScoresGetAllItemResponseDto } from "~/modules/quiz/quiz.js";
+
+const getCategoriesWithThreeLowestScores = (
+	categories: QuizScoresGetAllItemResponseDto[],
+): QuizScoresGetAllItemResponseDto[] => {
+	const categoriesSorted = categories.toSorted((a, b) => a.score - b.score);
```

Comment:
It is not clear what are "a" and "b". Let's use "leftCategory" and "rightCategory"